roadmap.rst

```diff
@@ -225,8 +225,8 @@ Date       Milestone
   
 - oneDAL
 
-  - Extend API description to 18 algorithms
-  - Add first version for sections on programming model, algorithmics
+  - Extend API description to kNN classification
+  - Add first version for sections on error handling, namespaces, terminology, math notations
```

Comment:
We can be flexible and do not tell about it, but as I understand we describe basic things related to data management (tables/accessors/builders) andd algorithms (descriptors, train etc)